lib/activity/router.js

```diff
@@ -20,8 +20,10 @@ module.exports = function route(callback) {
 
       const promise = Promise.all(subscriptions.map(async (subscription) => {
         if (!subscription.isEnabledForGitHubEvent(context.event)) {
+          context.log(`Checking subscription for event type: ${context.event}. (webhooks.subscription.match=0)`);
           return;
         }
+        context.log(`Checking subscription for event type: ${context.event}. (webhooks.subscription.match=1)`);
```

Comment:
I think we can also pass a hash at the beginning. So could do something like:
```suggestion
        context.log({"webhook_subscription_math": true},`Checking subscription for event type: ${context.event}. (webhooks.subscription.match=1)`);  
```

But totally up to you 💯

Reply:
Awesome. Using a hash is soooo much better than coming up with cryptic unique key-value patterns 👍